app/client/ui/UserManager.ts

```diff
@@ -816,14 +817,19 @@ const cssMemberPublicAccess = styled(cssMemberSecondary, `
 function renderTitle(resourceType: ResourceType, resource?: Resource, personal?: boolean) {
   switch (resourceType) {
     case 'organization': {
-      if (personal) { return t('Your role for this team site'); }
+      if (personal) {
+        return t('Your role for this team site');
+      }
+
+      const gristConfig = getGristConfig();
+      const gristHomeHost = gristConfig.homeUrl ? new URL(gristConfig.homeUrl).host : '';
+      const baseDomain = gristConfig.baseDomain || gristHomeHost;
+      const orgName = (resource as Organization).name;
```

Comment:
Later in the code you have `resource ? ... : ...` code fragment, which checks if `resource` is null or not, but here, you ignore this check and just grab the `.name` prop. Can you refactor it?

Reply:
How about this refactoring?